notifications_team_report/notifications_team_report.py

```diff
@@ -39,28 +39,28 @@ def main():
     ap.add_argument('-r', '--resume-file', dest='resume_file', default=False,
         type=argparse.FileType('rb'), help="Read previous ILE data from this\
         file instead of retrieving it from the API.")
-    ap.add_argument('-v', dest='verbosity', action='count', default=0,
-        help="Logging verbosity (default: INFO-level messages).")
+    ap.add_argument('-v','--verbose', help="Verbose command line output (show progress)",
+        default=False, action='store_true'     )
     ap.add_argument('-w', '--write-file', dest='write_file', default=False,
         type=argparse.FileType('wb'), help="A cache file to use for saving ILE\
         data that can be resumed from, before generating the reports. Useful\
         for if there's a massive quantity of ILE data to be saved, as a\
             safeguard in case something goes wrong in the reporting part.")
-    ap.add_argument('-t', '--token', type=str, help="A v2 REST API key to use\
-        for accessing notifications. A read-only key should suffice.")
-    args = ap.parse_args()
-    loglevs = ['info', 'critical', 'error', 'warning', 'info', 'debug']
-    loglev = min(args.verbosity, 5)
+    ap.add_argument('-k', '--api-key', type=str, required=True, 
+        dest='api_key', help="REST API key")
```

Comment:
Seemed like every other script in the repo uses `-k`, switched this for consistency.